tutor/templates/local/docker-compose.jobs.yml

```diff
@@ -25,6 +25,10 @@ services:
         - ../apps/openedx/settings/lms:/openedx/edx-platform/lms/envs/tutor:ro
         - ../apps/openedx/settings/cms:/openedx/edx-platform/cms/envs/tutor:ro
         - ../apps/openedx/config:/openedx/config:ro
-      depends_on: {{ [("mysql", RUN_MYSQL)]|list_if }}
+      depends_on:
+        {% if RUN_MYSQL %}- mysql{% endif %}
+        {% if RUN_ELASTICSEARCH %}- elasticsearch{% endif %}
+        {% if RUN_MONGODB %}- mongodb{% endif %}
+        {% if RUN_REDIS %}- redis{% endif %}
```

Comment:
Please use `list_if`to list all dependent services. Otherwise, the docker-compose file will have an incorrect syntax for people who don't run any of these services.